src/main/cli/cli.c

```diff
@@ -4924,7 +4924,7 @@ static void cliTasks(const char *cmdName, char *cmdline)
         taskInfo_t taskInfo;
         getTaskInfo(taskId, &taskInfo);
         if (taskInfo.isEnabled) {
-            int taskFrequency = taskInfo.averageDeltaTimeUs == 0 ? 0 : lrintf(1e6f / taskInfo.averageDeltaTimeUs);
```

Comment:
You can also improve accuracy by skipping truncation in `taskInfo->averageDeltaTimeUs = getTask(taskId)->movingSumDeltaTimeUs / TASK_STATS_MOVING_SUM_COUNT;`. Also float math is not necessary here.

```
int taskFrequency = taskInfo.movingSumDeltaTimeUs == 0 ? 0 : ((1000000 * TASK_STATS_MOVING_SUM_COUNT + taskInfo.movingSumDeltaTimeUs / 2) / taskInfo.movingSumDeltaTimeUs );

Reply:
@ledvinap I'd be honoured if you could join us in the Betaflight Slack group?